clap_man/src/render.rs

```diff
@@ -0,0 +1,195 @@
+use std::io::Write;
+
+use crate::man::{bold, document_title, escape, indent, italic, list, section_title};
+
+use clap::ArgSettings;
+
+fn app_name(app: &clap::App) -> String {
+    if let Some(name) = app.get_bin_name() {
+        escape(name)
+    } else {
+        escape(app.get_name())
+    }
+}
+
+fn subcommand_heading(app: &clap::App) -> String {
+    match app.get_subommand_help_heading() {
+        Some(title) => escape(title),
+        None => "Subcommands".to_string(),
+    }
+}
+
```

Comment:
Nit: I recommend putting the higher `pub` visibility items first since those are what lead people to read this file.

Reply:
Fixed.